.github/workflows/golanVE-ci.yml

```diff
@@ -24,24 +24,38 @@ jobs:
     - name: Test
       run: go test -v ./...
 
+    - name: Notify Telegram
+      uses: yanzay/notify-telegram@v0.1.0
+      if : always()
+      with:
+        chat: ${{ secrets.chat }}
+        token: ${{ secrets.token }}
+        status: ${{ job.status }}
+
   cd:
     runs-on: ubuntu-latest
-    needs: build
+    needs: ci
     
     steps:
-      - uses: actions/checkout@v3
-      - name: Docker login
-        run: docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
-      - name: Docker build
-        run: docker build -t adan-bot .
-      - name: Docker tags
-        run: |
-          docker tag adan-bot ${{ secrets.DOCKER_USER }}/adan-bot:${{ github.sha }}
-          docker tag adan-bot ${{ secrets.DOCKER_USER }}/adan-bot:latest
-      - name: Docker push
-        run: |
-          docker push ${{ secrets.DOCKER_USER }}/adan-bot:${{ github.sha }}
-          docker push ${{ secrets.DOCKER_USER }}/adan-bot:latest
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Docker build and push
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: |
+            ${{ secrets.DOCKER_USER }}/adan-bot:${{ github.sha }}
+            ${{ secrets.DOCKER_USER }}/adan-bot:latest
+    
```

Comment:
it's no necessary for this step, instead, change **Docker Registry** to **Github Registry**

Reply:
resolved.